core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/workspacemanager/WorkspaceManagerDAO.scala

```diff
@@ -3,34 +3,10 @@ package org.broadinstitute.dsde.rawls.dataaccess.workspacemanager
 import bio.terra.profile.model.ProfileModel
 import bio.terra.workspace.client.ApiException
 import bio.terra.workspace.model._
-import com.google.common.io.BaseEncoding.base64Url
 import org.broadinstitute.dsde.rawls.model.{DataReferenceDescriptionField, DataReferenceName, RawlsRequestContext}
 import org.broadinstitute.dsde.workbench.model.{ErrorReportSource, WorkbenchEmail}
 
-import java.nio.ByteBuffer
 import java.util.UUID
-import scala.util.Try
-
-object WorkspaceManagerDAO {
-
-  // Remove/Move into ShortUUID after [PF-1268]
-
-  /** @see bio.terra.stairway.ShortUUID#get() */
-  def decodeShortUuid(shortUuid: String): Option[UUID] =
```

Comment:
But code and unittests in https://broadworkbench.atlassian.net/browse/PF-1268?focusedCommentId=72596 in case needed again (didn't want to take the time to translate it to Java and submit to Stairway).